po/id.po

```diff
@@ -0,0 +1,475 @@
+"issues\n"
+"POT-Creation-Date: 2017-09-23 00:31+0800\n"
+"PO-Revision-Date: 2017-10-02 14:06+0700\n"
+"Last-Translator: Mahyuddin <mhy> <yudi.al@gmail.com>\n"
+"Language-Team: Indonesian (Indonesia) <yudi.al@gmail.com>\n"
+"Language: id_ID\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"X-Generator: Poedit 1.8.11\n"
+
+#: src/axel.c:89
+msgid "Buffer resized for this speed."
+msgstr "Buffer diubah ukurannya untuk kecepatan ini."
+
+#: src/axel.c:107
+msgid "Invalid URL"
+msgstr "URL tidak valid"
+
+#: src/axel.c:135
+msgid "Could not parse URL.\n"
+msgstr "Tidak bisa mengurai URL.\n"
+
+#: src/axel.c:157
+#, c-format
+msgid "Incomplete download found, ignoring no-clobber option\n"
+msgstr "Unduhan tidak lengkap ditemukan, mengabaikan opsi no-clobber\n"
+
+#: src/axel.c:160
+#, c-format
+msgid "File '%s' already there; not retrieving.\n"
+msgstr "Berkas '%s' sudah ada; tidak mengambil.\n"
+
+#: src/axel.c:191
+#, c-format
+msgid "File size: %lld bytes"
+msgstr "Ukuran berkas: %lld bytes"
+
+#: src/axel.c:220
+#, c-format
+msgid "Opening output file %s"
+msgstr "Membuka berkas keluaran %s"
+
+#: src/axel.c:228
+msgid "Server unsupported, starting from scratch with one connection."
+msgstr "Server tidak didukung, mulai dari dasar dengan satu koneksi."
+
+#: src/axel.c:245
+#, c-format
+msgid "%s.st: Error, truncated state file\n"
+msgstr "%s.st: Galat, bagian berkas terpotong\n"
+
+#: src/axel.c:253
+#, c-format
+msgid "Bogus number of connections stored in state file\n"
+msgstr "Jumlah koneksi palsu tersimpan dalam bagian berkas\n"
+
+#: src/axel.c:265
+#, c-format
+msgid "State file has old format.\n"
+msgstr "Bagian berkas memiliki format lama..\n"
+
+#: src/axel.c:298
+#, c-format
+msgid "State file found: %lld bytes downloaded, %lld to go."
+msgstr "Bagian berkas ditemukan: %lld bytes diunduh, %lld untuk berjalan."
+
+#: src/axel.c:304 src/axel.c:315
+msgid "Error opening local file"
+msgstr "Galat saat membuka berkas lokal"
+
+#: src/axel.c:328
+msgid "Crappy filesystem/OS.. Working around. :-("
+msgstr "Filesystem/OS jelek.. Bekerja di sekitar. :-("
+
+#: src/axel.c:342
+msgid "Error creating local file"
+msgstr "Galat membuat berkas lokal"
+
+#: src/axel.c:374
+#, c-format
+msgid ""
+"\n"
+"Reactivate connection %d\n"
+msgstr ""
+"\n"
+"Aktifkan kembali koneksi %d\n"
+
+#: src/axel.c:404
+msgid "Starting download"
+msgstr "Mulai mengunduh"
+
+#: src/axel.c:415 src/axel.c:598
+#, c-format
+msgid "Connection %i downloading from %s:%i using interface %s"
+msgstr "Sambungan %i diunduh dari %s:%i menggunakan antarmuka %s"
+
+#: src/axel.c:425 src/axel.c:610
+msgid "pthread error!!!"
+msgstr "pthread galat!!!"
+
+#: src/axel.c:470
+#, c-format
+msgid "Error while waiting for connection: %s"
+msgstr "Galat saat menunggu koneksi: %s"
+
+#: src/axel.c:502
+#, c-format
+msgid "Connection %i timed out"
+msgstr "Sambungan %ihabis waktunya"
```

Comment:
Is this string right? Seems to be missing a space after `%i`.

Reply:
I see, i will fix